UiTests/AnyOrgOrPersonalUiTest/AnyOrgOrPersonalTest.cs

```diff
@@ -0,0 +1,139 @@
+            string uriWithPort = TC.LocalhostUrl + ClientPort;
+
+            try
+            {
+                // Build the sample app with correct appsettings file.
+                UiTestHelpers.BuildSampleWithTestAppsettings(_testAssemblyLocation, _sampleAppPath, _testAppsettingsPath, SampleSlnFileName);
+
+                // Start the web app and api processes.
+                // The delay before starting client prevents transient devbox issue where the client fails to load the first time after rebuilding
+                var clientProcessOptions = new ProcessStartOptions(_testAssemblyLocation, _sampleAppPath, TC.s_oidcWebAppExe, clientEnvVars);
+
+                bool areProcessesRunning = UiTestHelpers.StartAndVerifyProcessesAreRunning([clientProcessOptions], out processes, NumProcessRetries);
+
+                if (!areProcessesRunning)
+                {
+                    _output.WriteLine($"Process not started after {NumProcessRetries} attempts.");
+                    StringBuilder runningProcesses = new StringBuilder();
+                    foreach (var process in processes)
+                    {
+#pragma warning disable CA1305 // Specify IFormatProvider
+                        runningProcesses.AppendLine($"Is {process.Key} running: {UiTestHelpers.ProcessIsAlive(process.Value)}");
+#pragma warning restore CA1305 // Specify IFormatProvider
+                    }
+                    Assert.Fail(TC.WebAppCrashedString + " " + runningProcesses.ToString());
+                }
+
+                LabResponse labResponse = await LabUserHelper.GetSpecificUserAsync(TC.OIDCUser);
+
+                // Initial sign in
+                _output.WriteLine("Starting web app sign-in flow.");
+                string email = labResponse.User.Upn;
+                await UiTestHelpers.NavigateToWebApp(uriWithPort, page);
+                await UiTestHelpers.EnterEmailAsync(page, email, _output);
+                await UiTestHelpers.EnterPasswordAsync(page, labResponse.User.GetOrFetchPassword(), _output);
+                await Assertions.Expect(page.GetByText("Integrating Azure AD V2")).ToBeVisibleAsync(_assertVisibleOptions);
+                await Assertions.Expect(page.GetByText(email)).ToBeVisibleAsync(_assertVisibleOptions);
+                _output.WriteLine("Web app sign-in flow successful.");
+
+                // Sign out
+                _output.WriteLine("Starting web app sign-out flow.");
+                await page.GetByRole(AriaRole.Link, new() { Name = "Sign out" }).ClickAsync();
+                await UiTestHelpers.PerformSignOut_MicrosoftIdFlow(page, email, TC.LocalhostUrl + ClientPort + SignOutPageUriPath, _output);
+                _output.WriteLine("Web app sign out successful.");
+            }
+            catch (Exception ex)
+            {
+                //Adding guid incase of multiple test runs. This will allow screenshots to be matched to their appropriet test runs.
```

Comment:
```suggestion
                // Adding guid in case of multiple test runs. This will allow screenshots to be matched to their appropriate test runs.
```